pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -227,42 +227,25 @@ in {
     latestCompatibleLinuxPackages = linuxPackages_6_1;
 
     # this package should point to the latest release.
-    version = "2.1.9";
+    version = "2.1.10";
 
-    sha256 = "RT2ijcXhdw5rbz1niDjrqg6G/uOjyrJiTlS4qijiWqc=";
+    sha256 = "sha256-ZREe6r7Z8JPj6dEmJesU1hVFwegx0J5CYeZ2b4draKY=";
```

Comment:
I think the latest compatible can be bumped to 6.2, right? [According to the release](https://github.com/openzfs/zfs/releases/tag/zfs-2.1.10)